extmod/modbtree.c

```diff
@@ -295,13 +295,16 @@ STATIC MP_DEFINE_CONST_DICT(btree_locals_dict, btree_locals_dict_table);
 STATIC const mp_obj_type_t btree_type = {
     { &mp_type_type },
     // Save on qstr's, reuse same as for module
+    .flags = MP_TYPE_FLAG_FULL,
```

Comment:
How about making `MP_TYPE_FLAG_FULL` and `EXTENDED_FIELDS` use the same nomenclature?
- `MP_TYPE_FLAG_EXTENDED` and `EXTENDED_FIELDS`
or
- `MP_TYPE_FLAG_FULL` and `FULL_FIELDS`
Then one would be more of a reminder for the other.

Reply:
To me, the option of calling the flag "extended" makes the most sense; changing the macro to FULL_FIELDS makes less sense to me. I can make that change if it's preferred.

Reply:
EXTENDED makes sense to me. thanks!

Reply:
Done!